torch/_torch_docs.py

```diff
@@ -3113,6 +3113,49 @@ def merge_dicts(*dicts):
 
 """)
 
+add_docstr(torch.inner, r"""
+inner(input, other, *, out=None) -> Tensor
+
+This functions computes the inner product of :attr:`input` and :attr:`other`
+if they are 1-D Tensors, otherwise computes a sum-product over the last dimensions.
+
+.. note:: 
+
+    If either input is a scalar, then a elemnt-wise multiplication is performed.
```

Comment:
If :attr:\`input\` or :attr:\`other\` is a scalar then this is equivalent to `torch.mul(a, b)`. If both :attr:\`input\` and :attr:\`other\` are tensors then their last (innermost) dimensions must have the same size.